### ISSUE TITLE
[Checkbox] hasLabel property hides the label on the wrong side in form + display

### ISSUE THREAD

**Issue by JRLK** [KleeGroup/focus-components#314]
![image](https://cloud.githubusercontent.com/assets/13674778/9815512/f049c12e-5896-11e5-9350-485dde9a6977.png)

**Comment by Tommass** [comment]
by using a checkbox, does it means the user have to chose "Contrat SVD17" = YES or NO ?
If yes, please display radio box with yes / no values.

**Comment by JRLK** [comment]
Yes, that's right the user have to chose Yes or No.
So I can be a radio box with a default value.
Can you tell me where I can find an example of radio box for a boolean field please?

Thanks

**Comment by Tommass** [comment]
Please now display boolean values (YES / NO) this way :

![image](https://cloud.githubusercontent.com/assets/5349745/9870275/ce496394-5b8a-11e5-9f2c-2e38267f2492.png)


Declare the the value in domain this way
```javascript
'DO_OUI_NON': {
        InputComponent: FocusComponents.common.select.radio.component,
        refContainer: {yesNoList: [{code: true, label: "select.yes"}, {code: false, label: "select.no"}]},
        listName: 'yesNoList'
    }
```

**Comment by Tommass** [comment]
Enhancement available in release v0.6.1.
